hacks/install_etcdctl

```diff
@@ -12,30 +12,47 @@ function show_help () {
 }
 
 function install () {
-  local version=$1
+  etcd_version="v3.4.26"
+  local version="${1:-${etcd_version}}"
   local download_url
+  local arch
+  local platform
+  local pkg
+  local dest
+  local yellow="\033[0;33m"
+  local nc="\033[0m"
+  local tmp_dir
 
-  if [ -z "$version" ]
-  then # fetch v3.4.26
-    version="v3.4.26"
-  else
-    if [[ ! $version == v* ]]
-    then
-      version="v${version}"
+  tmp_dir="$(mktemp -d)"
+  mkdir -p "${tmp_dir}/dest"
+
+  dest="/opt/bin/etcdctl"
+  if uname="$(whoami 2> /dev/null)"; then
+    if [[ "${uname}" == "root" ]]; then
+      if [ -w "/usr/local/bin" ]; then
+        dest="/usr/local/bin/etcdctl"
+      fi
     fi
   fi
 
-  arch=$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/')
+  arch=$(uname -m | sed 's/^x86_64$/amd64/;s/^aarch64$/arm64/')
 
-  echo "installing etcdctl version ${version}"
-  download_url="https://github.com/coreos/etcd/releases/download/${version}/etcd-${version}-linux-${arch}.tar.gz"
+  platform="$(uname -s | tr '[:upper:]' '[:lower:]')"
+  pkg="etcd-${version}-${platform}-${arch}"
 
-  curl -sL ${download_url} -o etcd-${version}-linux-${arch}.tar.gz && tar -zxvf etcd-${version}-linux-${arch}.tar.gz && mv etcd-${version}-linux-${arch}/etcdctl . && rm etcd-${version}-linux-${arch}.tar.gz && rm -r etcd-${version}-linux-${arch}
+  download_url="https://github.com/etcd-io/etcd/releases/download/${version}/${pkg}.tar.gz"
 
+  curl -sL "${download_url}" \
+    -o "${tmp_dir}/${pkg}.tar.gz" && \
+    tar -zxvf "${tmp_dir}/${pkg}.tar.gz" -C "${tmp_dir}/dest" && \
```

Comment:
Can we remove the `v` option from `tar`? We could probably replace it with an `echo`s that says that we are downloading it and then an echo that says that we are installing/unarchiving the app.
Same for the other tools provided via `install_on_demand`.

Reply:
Seems like the popular opinion. I will just remove the verbose parts.

Reply:
Verbose options are now removed for both `etcdctl` and `k9s`.